docs/deploy-configure.md

```diff
@@ -148,6 +148,10 @@ To define a custom role:
 
    | Key | Type | Description | Required? |
    |----|----|----|----|
+   | `spec.security.authentication.ignoreUnknownUsers` | boolean | 
+   Flag that indicates whether you can add users that do not exist 
+   in the `MongoDBCommunity` resource. If omitted, defaults to 
+   `true`. | No | 
```

Comment:
- remove the line breaks. github's markdown renderer handles these differently than our docs tooling. Click the kebab menu for this file on the `Files Changed` tab for this PR, then `View File` to see how this renders.

- do not -> don't ([per style guide)](https://docs.mongodb.com/meta/style-guide/style/contractions/)

```suggestion
   | `spec.security.authentication.ignoreUnknownUsers` | boolean | Flag that indicates whether you can add users that don't exist in the `MongoDBCommunity` resource. If omitted, defaults to `true`. | No | 
```

Reply:
also consider changing `MongoDBCommunity` resource to `MongoDB resource definition`. This term is used elsewhere in the docs.

Reply:
we renamed from `MongoDB` to `MongoDBCommunity` to avoid confusion with the enterprise `MongoDB` resource. I think if we are explicitly referring to the Kubernetes CR we can use MongoDBCommunity, what do you think?

Reply:
sounds good to me. We should make this change elsewhere in the docs, too.

Reply:
I'll open another ticket about the renaming.